src/Arcus.WebApi.Unit/Security/Authentication/SharedAccessKeyAuthenticationController.cs

```diff
@@ -15,5 +15,21 @@ public Task<IActionResult> TestHardCodedConfiguredSharedAccessKey(HttpRequestMes
         {
             return Task.FromResult<IActionResult>(Ok());
         }
+
+        [HttpGet]
+        [Route("authz/shared-access-key-header")]
```

Comment:
We can use the same constants you added previously for this. Thanks.

Reply:
done